src/Components/AddMedia/AddMedia.module.css

```diff
@@ -0,0 +1,29 @@
+.addfileButton {
+	width: 12vw;
```

Comment:
Buttons, icons etc. should be `px` rather than `vw`, otherwise they will change size on resize, which is an unexpected experience